src/lib/Scenes/Collection/Components/FeaturedArtists.tsx

```diff
@@ -0,0 +1,118 @@
+import { Box, Button, EntityHeader, Flex, Sans } from "@artsy/palette"
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
+import React from "react"
+import { TouchableHighlight, TouchableWithoutFeedback } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface FeaturedArtistsProps {
+  featuredArtists: any
+}
+
+interface FeaturedArtistsState {
+  showMore: boolean
+}
+
+export class FeaturedArtists extends React.Component<FeaturedArtistsProps, FeaturedArtistsState> {
+  state = {
+    showMore: false,
+  }
+
+  handleTap = (context: any, href: string) => {
+    SwitchBoard.presentNavigationViewController(context, href)
+  }
+
+  getFeaturedArtistEntityCollection = (artists: any) => {
+    return artists.map((artist, index) => {
+      const hasArtistMetaData = artist.nationality && artist.birthday
+      return (
+        <Box width="100%" key={index} pb={20}>
+          <TouchableWithoutFeedback onPress={() => this.handleTap(this, `/artist/${artist.slug}`)}>
+            <EntityHeader
+              imageUrl={artist.imageUrl}
+              name={artist.name}
+              meta={hasArtistMetaData ? `${artist.nationality}, b. ${artist.birthday}` : undefined}
+              href={`/artist/${artist.slug}`}
+              FollowButton={
+                <Button
+                  variant={artist.isfollowed ? "primaryBlack" : "secondaryOutline"}
+                  size="small"
+                  longestText="Following"
+                >
+                  {artist.isfollowed ? "Following" : "Follow"}
+                </Button>
+              }
+            />
+          </TouchableWithoutFeedback>
+        </Box>
+      )
+    })
+  }
+
+  render() {
+    const { featuredArtists } = this.props
+    const artists = featuredArtists.artworksConnection.merchandisableArtists
```

Comment:
This would be better if we could `import { get } from "lib/utils/get"` and then:

```const artists = get(this.props, p => p.featuredArtists.artworksConnection.merchandisableArtists)
const hasMultipleArtists = artists && artists.length > 1```

Reply:
[Updated](https://github.com/artsy/emission/pull/2001/files#diff-cf72bf49360aa2b6be4c6407c4b8ce68R56)! 

```
    const artists = get(this.props, p => p.collection.artworksConnection.merchandisableArtists)
    if (!artists) {
      return null
    }

    const hasMultipleArtists = artists.length > 1
```